posts/2021/10/hpy-0.0.4.md

```diff
@@ -0,0 +1,219 @@
+<!--
+.. title: hpy 0.0.4: Third public release
+.. slug: hpy-0.0.4-third-public-release
+.. date: 2021-10-22 15:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+HPy 0.0.4 is out! The third official HPy release comes with many new features
+and was again made available on PyPI (first time with binary wheels).
+
+Besides new API functions, the major highlights of this release are Python 3.10
+support, better support for native fields (HPyField) and global variables
+(HPyGlobal), 
```

Comment:
I would add a few more things to this list -- maybe mention one or two of the new API functions as examples and that there are new features in debug mode. Otherwise the introduction makes it sound like not too much happened when a lot did happen.